bee-network/bee-gossip/src/service/event.rs

```diff
@@ -100,6 +100,13 @@ pub enum Event {
         /// The peer's id.
         peer_id: PeerId,
     },
```

Comment:
```suggestion
    },
    
```
For consistency in this file even though it's not consistent with the rest of the project.

Reply:
Don't forget this please :)